source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/files/artifact.tsx

```diff
@@ -25,14 +25,19 @@ class Artifact extends React.Component<IProps> {
     const { artifactType, url, fileSize, dateUpdated } = attributesFor(artifact);
 
     return (
-      <div className='artifact-item flex'>
+      <div className='artifact-item flex p-l-md p-t-sm p-b-sm p-r-md'>
         <div className='flex-70 flex align-items-center'>
           <InsertDriveFileIcon className='m-r-sm' />
           <span data-test-project-files-artifact-name>{artifactType}</span>
         </div>
         <div className='flex-30 flex align-items-center justify-content-end'>
           <div className='position-relative w-40'>
-            {includeHeader ? <div className='column-header'>{t('project.products.updated')}</div> : ''}
+            {
+              includeHeader ?
+                <div className='column-header bold fs-13'>
```

Comment:
are none of these sizes sufficiently close?
![image](https://user-images.githubusercontent.com/199018/47575801-e46f3e80-d910-11e8-8913-4968bccd367b.png)